src/screens/StarRating.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Markdown, StarRating } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import {
+  ComponentDoc,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+} from '../components/Doc';
+
+const StarRatingPage = () => (
+  <Page>
+    <ComponentDoc
+      name="StarRating"
+      availableAt={[
+        {
+          url: 'https://storybook.grommet.io/?selectedKind=Visualizations-Spinner&full=0&stories=1&panelRight=0',
```

Comment:
```suggestion
          url: 'https://storybook.grommet.io/?selectedKind=Visualizations-Spinner&full=0&stories=1&panelRight=0',
```
Looks like this link needs updated

Reply:
done